### DISCUSSION TITLE
commands to setup Ubuntu firewall rules from the terminal

### DISCUSSION THREAD

**Discussion by xgithx** [telekom-security/tpotce#1670] [Q&A]
Which are the commands to configure Ubuntu firewall from the terminal, in order for the T-pot to work properly. Thank you.

**Comment by dmille6** [comment]
i've been fighting with the same thing.. i'd love to limit access to the web interface port and a few others, but cant seem to find an effective way to do it. if you figure it out please let me know.

**Comment by Ayaan7345** [comment]
Do you have a static ip that you can connect to your T-pot. Where is the T-pot placed such as in the cloud or self hosted? Try using ubuntu and then with ufw allow the web gui and the ssh and specify a specific ip. Or another option is dont expose those ports and then use tailscale vpn to connect to the machine.

**Comment by dmille6** [comment]
i'm self hosting with an ip. i've tried UFW . i've tried messing with bind.. no luck.. i ended up just putting it behind a virtual pfsense firewall, then only opening the ports/port forwarding the ports on the firewall I want exposed. There has to be a better way, but this way works.. but kind of a waste of resources
